build.gradle.kts

```diff
@@ -86,6 +86,27 @@ tasks.register("buildActivityApiModel", GenerateTask::class) {
   )
 }
 
+tasks.register("buildAdjudicationApiModel", GenerateTask::class) {
+  generatorName.set("kotlin")
+  skipValidateSpec.set(true)
```

Comment:
there swagger has some endpoints that we don't call that are invalid - so ignore them